docs/checkers/index.md

```diff
@@ -9,6 +9,34 @@ email to \[david at cs dot toronto dot edu\].
 These errors generally indicate a misuse of variables, control flow, or other Python features in our
 code.
 
+(W0134)=
+
+### Return in finally
```

Comment:
Oh, I missed this last time but (1) you should add the error code at the send of the heading (to be consistent with other headings), and (2) make sure this is categorized under "Exceptions" to make it go with the other exception-related errors.